pages/content/amp-dev/about/page-experience.html

```diff
@@ -3,6 +3,95 @@
 descriptions: Work in progress.
 $view: /views/2021/base.html
 ---
-<section class="ap-text-media">
-  <h1>Hello World</h1>
-</section>
+
+<main>
+  <section class="ap-section --col-4-15">
+    <div class="ap-text-block">
+      <h1>{{ _('Page experience success with AMP') }}</h1>
+      <p><a href="https://developers.google.com/search/docs/guides/page-experience">Page experience</a> is a set of signals from Google that measure how users perceive their experience interacting with a web page beyond its pure information value. <a href="https://ai.googleblog.com/2009/06/speed-matters.html">Google's internal studies</a> and <a href="https://blog.chromium.org/2020/05/the-science-behind-web-vitals.html">industry research</a> found that users prefer sites with a great page experience. A high page experience score helps your content surface to users in places like the <a href="https://support.google.com/news/publisher-center/answer/9607026?hl=en">top stories feature</a>, and encourages them to return to your site.</p>
+      <p><strong>AMP is a straightforward way to a high page experience score. </strong>Domains built with AMP are 5 times more likely to pass* <a href="https://web.dev/vitals/">Core Web Vitals</a> metrics than non-AMP!</p>
+    </div>
+    <a href="/page-experience" class="ap-button">{{ _('Check your page') }}</a>
```

Comment:
Use `href="{{g.doc('/content/amp-dev/page-experience.html', locale=doc.locale).url.path}}` for localized linking.